### ISSUE TITLE
Once `params` are merged into the spec, deprecate the `nonce` parameter

### ISSUE THREAD

**Issue by samuelgoto** [w3c-fedid/custom-requests#3]
> I would recommend removing the nonce parameter entirely in favor of moving all of that kind of thing into a different layer as described in w3c-fedid/FedCM#536.

This suggestion came up here https://github.com/fedidcg/FedCM/issues/582#issuecomment-2108559788, so kicking this issue off here to track it independently of the other issue.

This is a backwards incompatible change, so requires some thought put into coordinating with existing deployments, but I think is one that is worth pushing through.

Depends on w3c-fedid/continuation#2 .

**Comment by wseltzer** [comment]
Discussed at 11 April meeting [notes](https://github.com/w3c-fedid/meetings/blob/main/2025/2024-04-11-hybrid-notes.md).

**Comment by samuelgoto** [comment]
- immediately mark `nonce` deprecated in the spec
- asynchronously, work with existing IdPs that are deploying to move out of using `nonce` and use `params` instead.

**Comment by wseltzer** [comment]
Discussed [8 July](https://github.com/w3c-fedid/meetings/blob/main/2025/2025-07-08-FedCM-notes.md).

**Comment by pottis** [comment]
PR : https://github.com/w3c-fedid/FedCM/pull/762

**Comment by npm1** [comment]
Plz close since we landed the PR which removes nonce (can't close myself).
